redi-connect-front/src/components/atoms/Image.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Would be better to call it SVGImage, this way we now exactly what for this component is.

Reply:
renamed